enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metabase/embedding-sdk-react",
-  "version": "1.52.1",
+  "version": "0.52.1-nightly",
```

Comment:
We don't actually have this version, but this version will be used by the workflow to create a new version. And from the test run it looks like everything works as expected. T[he changelog](https://github.com/metabase/metabase/pull/49884) also looks correct.